qadence/ml_tools/train_grad.py

```diff
@@ -61,6 +66,10 @@ def train(
             called every `config.write_every` iterations. The function must have
             the signature `write_tensorboard(writer, loss, metrics, iteration)`
             (see the example below).
+        epsilon: Safety margin to check if validation loss is smaller than the lowest
```

Comment:
can we move this into the `TrainConfig` rather?

Reply:
Yes, done, thanks!